crates/test_utils/src/newman_runner.rs

```diff
@@ -0,0 +1,155 @@
+    newman_command.args(["--env-var", &format!("baseUrl={base_url}")]);
+
+    if let Some(auth_type) = inner_map.get(&connector_name) {
+        match auth_type {
+            ConnectorAuthType::HeaderKey { api_key } => {
+                newman_command.args([
+                    "--env-var",
+                    &format!("connector_api_key={}", api_key.peek()),
+                ]);
+            }
+            ConnectorAuthType::BodyKey { api_key, key1 } => {
+                newman_command.args([
+                    "--env-var",
+                    &format!("connector_api_key={}", api_key.peek()),
+                    "--env-var",
+                    &format!("connector_key1={}", key1.peek()),
+                ]);
+            }
+            ConnectorAuthType::SignatureKey {
+                api_key,
+                key1,
+                api_secret,
+            } => {
+                newman_command.args([
+                    "--env-var",
+                    &format!("connector_api_key={}", api_key.peek()),
+                    "--env-var",
+                    &format!("connector_key1={}", key1.peek()),
+                    "--env-var",
+                    &format!("connector_api_secret={}", api_secret.peek()),
+                ]);
+            }
+            ConnectorAuthType::MultiAuthKey {
+                api_key,
+                key1,
+                key2,
+                api_secret,
+            } => {
+                newman_command.args([
+                    "--env-var",
+                    &format!("connector_api_key={}", api_key.peek()),
+                    "--env-var",
+                    &format!("connector_key1={}", key1.peek()),
+                    "--env-var",
+                    &format!("connector_key2={}", key2.peek()),
+                    "--env-var",
+                    &format!("connector_api_secret={}", api_secret.peek()),
+                ]);
+            }
+            // Handle other ConnectorAuthType variants
+            _ => {
+                eprintln!("Invalid authentication type.");
+            }
+        }
+    } else {
+        eprintln!("Connector not found.");
+    }
+
+    // Add additional environment variables if present
+    if let Ok(gateway_merchant_id) = env::var("GATEWAY_MERCHANT_ID") {
+        newman_command.args([
+            "--env-var",
+            &format!("gateway_merchant_id={gateway_merchant_id}"),
+        ]);
+    }
+
+    if let Ok(gpay_certificate) = env::var("GPAY_CERTIFICATE") {
+        newman_command.args(["--env-var", &format!("certificate={gpay_certificate}")]);
+    }
+
+    if let Ok(gpay_certificate_keys) = env::var("GPAY_CERTIFICATE_KEYS") {
+        newman_command.args([
+            "--env-var",
+            &format!("certificate_keys={gpay_certificate_keys}"),
+        ]);
+    }
+
+    newman_command.arg("--delay-request").arg("7"); // 7 milli seconds delay
+
+    newman_command.arg("--color").arg("on");
```

Comment:
It will be a good idea to expose all postman run options via this CLI - might be useful at some time.  things like `delay-request` and `timeout` would be useful.

Reply:
Sure! I'll raise another PR soon addressing this since this is already merged